test/number.spec.ts

```diff
@@ -0,0 +1,333 @@
+      });
+    });
+
+    describe('hex', () => {
+      it('generates single hex character when no additional argument was provided', () => {
+        const hex = faker.number.hex();
+        expect(hex).toBeTypeOf('string');
+        expect(hex).toHaveLength(1);
+        expect(hex).toSatisfy(validator.isHexadecimal);
+      });
+
+      it('generates a random hex string', () => {
+        const hex = faker.number.hex(5);
+        expect(hex).toSatisfy(validator.isHexadecimal);
+      });
+
+      it('generates a random hex in a specific range', () => {
+        const hex = faker.number.hex({ min: 15, max: 255 });
+
+        const hexNum = parseInt(hex, 16);
+        expect(hexNum).toBeLessThanOrEqual(255);
+        expect(hexNum).greaterThanOrEqual(15);
+      });
+
+      it('should throw when min > max', () => {
+        const min = 10;
+        const max = 9;
+
+        expect(() => {
+          faker.number.hex({ min, max });
+        }).toThrowError(`Max ${max} should be greater than min ${min}.`);
+      });
+    });
+
+    describe('bigInt', () => {
+      it('should generate a bigInt value', () => {
+        const generateBigInt = faker.number.bigInt();
+        expect(generateBigInt).toBeTypeOf('bigint');
+      });
+
+      it('should generate a big bigInt value with low delta', () => {
+        const generateBigInt = faker.number.bigInt({
+          min: 999999999n,
+          max: 1000000000n,
+        });
+        expect(generateBigInt).toBeTypeOf('bigint');
+        expect(generateBigInt).toBeGreaterThanOrEqual(999999999n);
+        expect(generateBigInt).toBeLessThanOrEqual(1000000000n);
+      });
+
+      it('should return a random bigint given a maximum value as BigInt', () => {
+        expect(faker.number.bigInt(10n)).toBeGreaterThanOrEqual(0n);
+        expect(faker.number.bigInt(10n)).toBeLessThanOrEqual(10n);
+      });
+
+      it('should return a random bigint given a maximum value as Object', () => {
+        expect(faker.number.bigInt({ max: 10n })).toBeGreaterThanOrEqual(0n);
+        expect(faker.number.bigInt({ max: 10n })).toBeLessThanOrEqual(10n);
+      });
+
+      it('should return a random bigint given a maximum value of 0', () => {
+        expect(faker.number.bigInt({ max: 0n })).toBe(0n);
+      });
+
+      it('should return a random bigint given a negative bigint minimum and maximum value of 0', () => {
+        expect(
+          faker.number.bigInt({ min: -100n, max: 0n })
+        ).toBeGreaterThanOrEqual(-100n);
+        expect(
+          faker.number.bigInt({ min: -100n, max: 0n })
+        ).toBeLessThanOrEqual(0n);
+      });
+
+      it('should return a random bigint between a range', () => {
+        const randomBigInt = faker.number.bigInt({ min: 22, max: 33 });
+        expect(randomBigInt).toBeGreaterThanOrEqual(22);
+        expect(randomBigInt).toBeLessThanOrEqual(33);
+      });
+
+      it('should succeed with success-rate', () => {
```

Comment:
```suggestion
      it('should succeed with success-rate', () => {
```
What does 'succeed with success-rate' mean?

Reply:
I don't know :thinking: 
Somehow it feels like we can just remove this test, can we?